src/pages/Embeds/Plio.vue

```diff
@@ -670,6 +746,10 @@ export default {
       // convert retention array to retention string
       return retentionArray.join(",");
     },
+    setScreenProperties() {
+      // sets various properties based on the device screen
+      this.setPlayerAspectRatio();
+    },
```

Comment:
best to remove this function now and just keep setPlayerAspectRatio
If we have some more screen related functions later on then we can do this

because anyway we're selectively using `setPlayerAspectRatio` for example in the `playerInitiated` method